localPOA.json

```diff
@@ -6,6 +6,7 @@
     "eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
     "eip155Block": 3,
     "eip158Block": 3,
+    "byzantiumBlock": 4,
```

Comment:
I'm not sure if we'd ever deploy contracts in blocks 1-3, but would geth reject the ones that used byzantium opcodes if so? Probably a problem we can work around.

Reply:
Geth wouldn't reject the deployment, but transactions that call functions in the contracts that use Byzantium opcodes would fail with an invalid opcode error before the Byzantium block. I think the block number chosen is arbitrary, but I don't think it would present a problem as long as its one of the first few blocks